SwrveSDK/src/main/java/com/swrve/sdk/SwrveBase.java

```diff
@@ -684,7 +685,13 @@ public void onResponse(RESTResponse response) {
                                 }
 
                                 try {
-                                    JSONObject responseJson = new JSONObject(response.responseBody);
+                                    JSONObject responseJson = null;
+                                    try {
+                                        responseJson = new JSONObject(response.responseBody);
+                                    } catch(JsonSyntaxException e) {
+                                        SwrveLogger.e("SwrveSDK unable to decode \"" + response.responseBody + "\" with JSON.");
```

Comment:
Might be useful to log:
"SwrveSDK unable to decode user_resources_and_campaigns JSON : \"" + response.responseBody